packages/gatsby-telemetry/src/get-dependencies.ts

```diff
@@ -0,0 +1,39 @@
+import { readFileSync } from "fs"
```

Comment:
we use fs-extra for better os support
```suggestion
import { readFileSync } from "fs-extra"
```